CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ project(mixxx VERSION 2.3.0)
 set(CMAKE_PROJECT_HOMEPAGE_URL "https://www.mixxx.org")
 set(CMAKE_PROJECT_DESCRIPTION "Mixxx is Free DJ software that gives you everything you need to perform live mixes.")
 
+# Compiler flag to get Google Test coverage statistics
+set(CMAKE_CXX_FLAGS --coverage)
```

Comment:
I think this should be added to the `target_compile_options` in line 945. Do we need a flag for this? Has it any impact on build time or speed?

Reply:
Have not profiled the performance yet.
The necessity of this flag would be decided by the community.
Link to the corresponding conversation:
[https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Code.20Coverage/near/202995543](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Code.20Coverage/near/202995543)